src/pf-modal/index.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+<head>
+    <title>pf-tooltip example</title>
+    <link rel="stylesheet" href="/dist/css/patternfly.css">
```

Comment:
you'll want to make all of these `dist/css/` paths relative after the recent update. i.e. `../../dist/css/patternfly.css`. Check the other components for an example on this...